packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -192,9 +192,13 @@ const pincodeSetScreenOptions = ({
   route: RouteProp<StackParamList, Screens.PincodeSet>
 }) => {
   const isVerifying = route.params?.isVerifying
+  const changePin = route.params?.changePin
   const title = isVerifying
     ? i18n.t('onboarding:pincodeSet.verify')
+    : changePin
+    ? i18n.t('onboarding:pincodeSet.changePIN')
     : i18n.t('onboarding:pincodeSet.create')
+
   return {
     ...nuxNavigationOptions,
     headerTitle: () => (
```

Comment:
still need to update this with an empty subtitle but having trouble with using an empty string here